bw2io/export/gexf.py

```diff
@@ -130,15 +223,35 @@ def __init__(self, database, keys):
 
 
 def keyword_to_gephi_graph(database, keyword):
-    """Export a Gephi graph for a database for all activities whose names include the string ``keyword``.
+    # """Export a Gephi graph for a database for all activities whose names include the string ``keyword``.
```

Comment:
...same question here: [why are you keeping the old docstrings in and commenting them out?](https://github.com/brightway-lca/brightway2-io/pull/169/files#r1141269894)